### PR TITLE
Docs/update documentation: Update dpp documentation with current implementation

### PR DESCRIPTION
**Pull request by saudkhan116**
# Why we create this PR?
 
To update the documentation to align with current implementation.
 
# What we want to achieve with this PR?
 
The DPP documentation needs to be updated and shoud align with the current implemention.
 
# What is new?
 
## Updated

- GETTING-STARTED.md
- RELEASE.md
- SECRETS_MANAGEMENT.md
- Arc42.md
- Admin_Guide.md
- deployment/readme
- Navigation links in the root readme
- User Manual Product Viewer App.md
- Consumer-backend readme

### CODE REVIEW

**Comment by matbmoser** [comment]
This branch will be rebased in tractus-x main

**Comment by matbmoser** [comment]
Most of the changes are related to documentation. 
This is the reason why there is so much files changed

**Comment by matbmoser** [comment]
This PR contains the PRs #22, #23 and #26. When a PR is merged to develop all the changes go into the branch because developed gets merged to resolved conflicts in the branch

**Comment by matbmoser** [comment]
The conflicts need to be resolved.

**Comment by matbmoser** [comment]
Is necessary to add the change logs for the version v0.7.0, upgrade the helm charts version to the new release and fix minor bug in helm chart related with the kics workflow: https://github.com/eclipse-tractusx/digital-product-pass/actions/runs/4938855741/jobs/8829072017#step:4:2012

File: consumer-backend/productpass/pom.xml
Comment (matbmoser) [inline consumer-backend/productpass/pom.xml:1]: The backend version was increased. This was a bug which  needed to be fixed from a documentation perspective.

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,7 +23,39 @@
...
+- Prepare environment-specific values.yaml
+- Prepare values.yaml for default mode
+- Added Tooltip Component
+- Added Bar Component Fix
+- Added new Readme to guide the developer/user thought the documentation
+  
+## Security Fixes
+- Blocked accessExcalation in the frontend helm templates.
 
 ## [in preparation]
```
Comment (matbmoser) [inline CHANGELOG.md]: here the readme is duplicated requires a fix.

Comment (matbmoser) [reply CHANGELOG.md]: I fixed by my self so its done  👍🏻

**Review by matbmoser — APPROVED** [review]
From a content perspective it approved.

Most of the changes are **documentation** updates. This is the reason for 55 changed files.

In general now the quality of the documentation has really increased thanks to this PR.

After this PR we can create a new release v0.7.0 using the changelog description.

**Review by SebastianBezold — APPROVED** [review]

**Comment by SebastianBezold** [comment]
Hi @matbmoser, 
thanks for reviewing the content. Question about creating a release? The Changelog suggests, that a `v0.7.0` should be created. In your backend `pom.xml`, `0.5.0` is mentioned.
I guess this is just for the backend component, but the overall version for the release should be `0.7.0` right?

**Comment by matbmoser** [comment]
> Hi @matbmoser, thanks for reviewing the content. Question about creating a release? The Changelog suggests, that a `v0.7.0` should be created. In your backend `pom.xml`, `0.5.0` is mentioned. I guess this is just for the backend component, but the overall version for the release should be `0.7.0` right?

Thank you @SebastianBezold for merging this pull request, yes we need to create the release and as content the changelog from the version v0.7.0. 

You are right the backend version has nothing to do with the actual version from the complete application, it just keeps track of the backend component updates, since it was created after the actual application was developed.

**Pull request merged**